scriptworker.yaml.tmpl

```diff
@@ -0,0 +1,145 @@
+#-----------------------------------------------------------------------------------------------
+# Taskcluster worker settings.
+# For development, keep the test-dummy-* and dummy-worker-* convention.
+# In production, follow the production conventions.
+#-----------------------------------------------------------------------------------------------
+provisioner_id: test-dummy-provisioner
+worker_group: test-dummy-workers
+worker_type: dummy-worker-myname
+
+# worker_id will default to env['SCRIPTWORKER_WORKER_ID'] if not set, for CloudOps deployment.
+# We may be able to remove that default if we don't use CloudOps deployment for scriptworker
+# instances.
+worker_id: dummy-worker-myname1
+
+
+#-----------------------------------------------------------------------------------------------
+# Taskcluster credentials.
+# Uncomment and edit to specify the taskcluster credentials here.
+# Taskcluster credentials can also be set in secrets.json, $HOME/.scriptworker,
+# /etc/.scriptworker, or the environment variables TASKCLUSTER_ACCESS_TOKEN,
+# TASKCLUSTER_CLIENT_ID, and TASKCLUSTER_CERITIFICATE.
+#-----------------------------------------------------------------------------------------------
+# credentials:
+#    clientId: ...
+#    accessToken: ...
+#    certificate: ...
+
+#-----------------------------------------------------------------------------------------------
+# Task configs
+#-----------------------------------------------------------------------------------------------
+artifact_expiration_hours: 24
```

Comment:
Out of curiosity, would it worth to nest the task configuration? For example:
```yaml
task:
  artifact_expiration_hours: 24
  max_timeout: 1200
```
or 
```yaml
gpg:
  email: "scriptworker@example.com"
```

Edit: After going further in the PR, I realized this'll break too many things.